tests/test_anisotropic_noise.py

```diff
@@ -120,8 +121,31 @@ def testAnisotropicNoiseVariance(self):
         noise_variance = noise_estimator.estimate()
         self.assertTrue(
             np.allclose(
-                0.04534846544265747,
+                0.005158715099241817,
                 noise_variance,
                 atol=utest_tolerance(self.sim.dtype),
             )
         )
+
+    def testPerseval(self):
+        """
+        Here we construct a source of white noise.
+        Then code tests that the average noise power in the real domain,
+        is equivalent to the sum of the magnitudes squared
+        of all frequency's coefficients in the Fourier domain.
+
+        These are calculated by WhiteNoiseEstimator and
+        AnisotropicNoiseEstimator respectively.
+
+        See Perseval/Plancherel's Theorem.
```

Comment:
Parseval I'm 99% sure 😛

Reply:
yes, you are right, I was rushing. Will change the spelling mistakes. Thanks